src/presentation/components/NeogaFormImageToSave/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import NeogaFormTicket from '@components/NeogaFormTicket';
+import { StLogo, StNeogaFormImageToSave } from './style';
+import { NeososeoFormData } from '@api/types/neososeo-form';
+import { imgCharacterLogo, imgEmptyProfile } from '@assets/images';
+
+interface NeogaFormImageToSaveProps {
+  formData: NeososeoFormData;
+}
+
+export const NeogaFormImageToSave = React.forwardRef<HTMLDivElement, NeogaFormImageToSaveProps>(
+  (props, ref) => {
+    const { title, content, imageSub, userName, createdAt, userProfileImage } = props.formData;
+    return (
+      <StNeogaFormImageToSave ref={ref}>
+        <div>
+          <img src={userProfileImage?.length ? userProfileImage : imgEmptyProfile} />
```

Comment:
```suggestion
          <img src={userProfileImage ? userProfileImage : imgEmptyProfile} />
```
이렇게 작성하지 않고 length를 명시한 이유가 있을까용?

Reply:
프로필 이미지가 없을 때 빈 string으로 오는 경우가 있어서요 지금 이미지 상태가 다소 중구난방인데 한 번 전반적으로 통일하면 좋을 것 같긴 합니다 ..